KalturaWowzaServer/src/main/java/com/kaltura/media/server/wowza/listeners/LiveStreamEntry.java

```diff
@@ -204,6 +218,8 @@ public void onCodecInfoVideo(IMediaStream stream, MediaCodecInfoVideo videoCodec
 		public void onPublish(IMediaStream stream, String streamName, boolean isRecord, boolean isAppend) {
 
 			logger.debug("Stream [" + streamName + "]");
+            String entryId = "";
+            KalturaMediaServerIndex serverIndex = KalturaMediaServerIndex.PRIMARY;
```

Comment:
the IDE is mad about u here, it says that setting serverIndex to PRIMARY is redundant.
read the IDE message and fix it, it will also make your code more readable